libpod/runtime_img.go

```diff
@@ -93,7 +94,11 @@ func (r *Runtime) RemoveImage(ctx context.Context, img *image.Image, force bool)
 			err = errStorage
 		}
 	}
-	return img.ID(), err
+	if len(img.Names()) == 1 {
+		returnMessage = fmt.Sprintf("Untagged: %s\n", img.Names()[0])
```

Comment:
Should we just `for` loop this?
```
returnMessage := ""
for _, name := img.Names() {
     returnMessage = returnMessage + fmt.Sprintf("Untagged: %s\n", name)
}
returnMessage = returnMessage + fmt.Sprintf("Deleted: %s", img.ID())
```

Reply:
No, but it does bring up an interesting edge case ... lemme go test

Reply:
yeah, added that change